### ISSUE TITLE
When playing IPTV Streams with exteplayer3, the web interface improperly displays Cyrillic characters as percent-encoded strings.

### ISSUE THREAD

**Issue by Box333** [oe-alliance/OpenWebif#40]
This bug occurs in the `Classic` web interface.
This bug occurs when playing IPTV Streams with exteplayer3 interfaces.

The web interface improperly displays Cyrillic characters as percent-encoded strings e.g.:
"1+1 %D0%A3%D0%BA%D1%80%D0%B0%D1%97%D0%BD%D0%B0 HD" 
instead of rendering the actual name:
 1+1 Україна HD

**Image (please complete the following information):**
 - OS: OpenATV 
 - Version 7.x and before

**Desktop (please complete the following information):**
 - Browser: all

**Comment by jbleyel** [comment]
I can't reproduce this.
The problem seams to be in your userbouquet file.
Maybe the encoding of the file is not UTF-8

**Comment by Box333** [comment]
no, the userbouquet is fine, this happens only on servicetype 5002. 
Here you see the improperly displayed channel name for the same channel set up with gststreamer and exteplayer3.

![Webif](https://github.com/user-attachments/assets/2575fc1f-2700-47c5-b376-664e6a334ddf)
![Webif1](https://github.com/user-attachments/assets/a861ce98-1809-43d1-9a71-f5518708f566)

**Comment by jbleyel** [comment]
Can you please provide a bouquet file with one problem channel?

**Comment by Box333** [comment]
here are the 2 entries from the *.tv bouquet file, which are behind/represent the above screenshots:

#SERVICE 5002:0:1:6294:e3a8:d17:dddd0000:0:0:0:http%3a//iptv.xyz/iptv/2402/index.m3u8#tvg-id=146:Первый канал FHD
#DESCRIPTION Первый канал FHD
#SERVICE 5002:0:1:40a2:1042:d17:dddd0000:0:0:0:http%3a//iptv.xyz/iptv/2409/index.m3u8#tvg-id=711&:Россия 1 FHD
#DESCRIPTION Россия 1 FHD

**Comment by Box333** [comment]
you fixed it 👍 Thanks a lot !
